data/json/items/vehicle/animals.json

```diff
@@ -25,7 +25,7 @@
     "category": "veh_parts",
     "description": "Heavy steel beams and harness to attach a sufficiently large creature to a wheeled vehicle, allowing them to pull it along, reinforced to withstand the rigors of the apocalypse and ill-advised off-roading.  You would still need separate armor for said creature to wear.",
     "weight": "35000 g",
-    "//": "calculating weight: yoke_harness (1000g) + frame (20 000g) + scrap (50g)x35 OR steel_lump(1000g)x5 + pipe(1250g)x2 + chain(3628)x2. Total 1: 32506g. Total 2: 35756g. With rounding and fudge factor: 35000g. Previus eyeballed guess before doing any of this math was 30000g.",
+    "//": "calculating weight: yoke_harness (1000 g) + frame (20 000 g) + scrap (50 g)x35 OR steel_lump(1000 g)x5 + pipe(1250 g)x2 + chain(3628)x2. Total 1: 32506 g. Total 2: 35756 g. With rounding and fudge factor: 35000 g. Previus eyeballed guess before doing any of this math was 30000 g.",
```

Comment:
I don't see the point in doing this in non user-facing strings.

Reply:
I did it for consistency and felt like it improved readability in some places, especially where there were things like ```53cmx47cmx~3cm``` were happening, but I can do it only for strings the user is going to see if that's what you think is best.

Reply:
I am not sure the readability is worth the increased churn and difficulty in review, and think it would be best to keep it to user-facing strings. But I have also added some of the comments being changed :)

Reply:
I'll refrain from making changes to comments in the follow-up PRs. Do you want me to revert the changes made to comments in this PR or is it alright if I leave them?

Reply:
I do not think it is worth your time to go through and manually delete spaces.